src/azure-cli/azure/cli/command_modules/rdbms/_helptext_pg.py

```diff
@@ -381,7 +381,7 @@
 
 helps['postgres flexible-server firewall-rule list'] = """
 type: command
-short-summary: List all firewall rules for a flexible server.
+short-summary: List all firewall rules for a flexible server. Run the command with `--debug` switch to identify the REST API that is invoked underneath and the response received from the service.
```

Comment:
`--debug` is a global parameter, not only for this command. There's no need to add a instruction here

Reply:
@evelyn-ys The OP of this thread https://github.com/Azure/azure-cli/issues/27463#issuecomment-1736889096  requested it to be added. Hence the PR was filed:

<img width="551" alt="image" src="https://github.com/Azure/azure-cli/assets/57353862/951589b6-2430-4831-9a70-4c15ace180e7">

Reply:
Please share https://learn.microsoft.com/en-us/cli/azure/get-started-with-azure-cli#globally-available-arguments with customer